From e54df5f611410d35b86a6d1c02068c7852f734f9 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:07 -0400
Subject: [redhat] kabi: Adding symbol mempool_free_slab (mm/mempool.c)

Message-id: <f89ccdf59adadd2e4af6d7349d47d401556617b5.1410510773.git.poros@redhat.com>
Patchwork-id: 96102
O-Subject: [PATCH RHEL7.1 127/233] kabi: Adding symbol mempool_free_slab (mm/mempool.c)
Bugzilla: 1062361 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1062361,1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Using kernels default slab deallocator function passed to mempool create.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/mempool_free_slab
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/mempool_free_slab
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/mempool_free_slab

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/mempool_free_slab b/redhat/kabi/kabi-module/kabi_ppc64/mempool_free_slab
new file mode 100644
index 0000000..2f1f46a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/mempool_free_slab
@@ -0,0 +1,2 @@
+#1-
+0x8a99a016 mempool_free_slab vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/mempool_free_slab b/redhat/kabi/kabi-module/kabi_s390x/mempool_free_slab
new file mode 100644
index 0000000..2f1f46a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/mempool_free_slab
@@ -0,0 +1,2 @@
+#1-
+0x8a99a016 mempool_free_slab vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/mempool_free_slab b/redhat/kabi/kabi-module/kabi_x86_64/mempool_free_slab
new file mode 100644
index 0000000..2f1f46a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/mempool_free_slab
@@ -0,0 +1,2 @@
+#1-
+0x8a99a016 mempool_free_slab vmlinux EXPORT_SYMBOL
-- 
1.7.1